src/StatusQ/Components/StatusClickableTag.qml

```diff
@@ -0,0 +1,139 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+/*!
+   \qmltype StatusAccountSelectorTag
+   \inherits Rectangle
+   \inqmlmodule StatusQ.Components
+   \since StatusQ.Components 0.1
+   \brief The StatusAccountSelectorTag provides a template for creating a clickable tag item.
+
+   Example of how to use it:
+
+   \qml
+        StatusClickableTag {
+            title: "some name"
+            icon.name: "some icon name"
+            isSelected: false
+            onClicked: {
+                // do something
+            }
+        }
+   \endqml
+
+   For a list of components available see StatusQ.
+*/
+Rectangle {
+    id: root
+
+    /*!
+       \qmlproperty titleText
+        This property represents the Text component which loads the text in the tag
+        Can be used to customise the Text component in the tag
+        \endqml
+    */
+    property alias titleText: titleText
+    /*!
+       \qmlproperty title
+        This property represents the title in the tag
+        Can be used to assign text to the tag
+        \endqml
+    */
+    property string title: ""
+    /*!
+       \qmlproperty isSelected
+        This property represents if the tag should be highlighted
+        \endqml
+    */
+    property  bool isSelected: false
+    /*!
+       \qmlproperty maxTextWidth
+        This property represents the max widht allowed for the title after which it will elide
+        \endqml
+    */
+    property int maxTextWidth: 93
```

Comment:
Not that it makes a huge difference, but would it be okay to use some even value instead?